src/get-location.js

```diff
@@ -0,0 +1,5 @@
+const getLocation = () => {
```

Comment:
super minor: you could probably put this in the `utils` file instead of making a new module for it